test/convergence.jl

```diff
@@ -75,18 +75,16 @@ function tabulate_convergence_orders_imex_ark()
     ]
     tabs = map(t -> t(), tabs)
     test_cases = all_test_cases(Float64)
-    results = convergence_order_results(tabs, test_cases)
-    algs = algorithm.(tabs)
-    prob_names = map(t -> t.test_name, test_cases)
-    expected_orders = ODE.alg_order.(tabs)
-    tabulate_convergence_orders(prob_names, algs, results, expected_orders; tabs)
+    get_alg = (tab, test_case) -> IMEXARKAlgorithm(tab, NewtonsMethod(; max_iters = test_case.linear_implicit ? 1 : 2))
```

Comment:
Why is `max_iter` necessary for linear implicit problems? Won't the alg just converge early even if `max_iters = 2`?

Reply:
Also, one thing I don't like about this is that it couples the rows and columns of the tables-- the algo used is a function of the test case-- and I prefer that we explicitly prescribe them instead, independent of the test problem.

Reply:
In my initial implementation, `max_iter` was always 2 by default, and there was a bug in Newton's method which was not caught for over a month because of that. If `max_iter` had been 1 for linear problems, the bug would have been caught. We specifically need to test that Newton's method works correctly with a single iteration for linear problems. If you don't like the rows and columns being coupled, we could distinct tables for linear and non-linear problems.

Reply:
Per Tapio from a few months ago, we need to see the correct convergence of linear problems with a single iteration. So it should in fact be part of the convergence sweep.